src/org/omegat/filters4/xml/xliff/Xliff1Filter.java

```diff
@@ -60,6 +60,11 @@ protected final String versionPrefix() {
         return "1."; // can be 1.0, 1.1 or 1.2
     }
 
+    @Override
+    public boolean isBilingual() {
```

Comment:
You can put it in AbstractXliffFilter instead, I imagine that if later there is an XLIFF 3 it will be bilingual as well